src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -71,6 +71,13 @@ public GenericMeasureInfo()
         ///
         /// </summary>
         public double? Minimum { get; set; }
+
+        /// <summary>
+        ///
```

Comment:
Please remove such extra lines.

Reply:
We're talking about the empty comment lines like line 76 right?

Reply:
Also, do we want to remove lines like line 378 where there is no summary included?

Reply:
Yes, please remove empty comment lines in your code. We usually put comment right under line where we see a problem.
The empty line in public comment gets into auto-generated documentation and corrupts its formatting.
The 378 line is not your code - you shouldn't correct it until reviewer/maintainer ask you. Usually we open new tracking Issue for changing bad patterns and fix its in separate PR/commit.